packages/components/src/Card/Card.tsx

```diff
@@ -44,6 +46,9 @@ export function Card({
     accent && colors[accent],
   );
 
+  const tagProps = {
+    ...(external && { target: "_blank" }),
+  };
```

Comment:
You can actually write this on the element itself on line `78`. Saves you a const and also tagProps only applies to the `a` tag so that also makes it straightforward

```jsx
<a className={className} target={external ? "_blank" : "_self"} href={url}>

Reply:
Which exposes this issue. 
![image](https://user-images.githubusercontent.com/15986172/136058488-fae5c204-f5e2-4e07-80a5-88f2e05ed5e1.png)

Reply:
@darryltec ahh, thanks for catching that! 

@shreyavi, in that case could you add the `rel` attribute as mentioned in the screenshot above if the target is `_blank`

Reply:
@darryltec in that case what do I put in else for rel here       
`<a className={className} target={external ? "_blank" : "_self"} rel={external ? "noopener noreferrer"} href={url}>
`
Any ideas?

Reply:
tag makes the most sense imo, https://www.w3schools.com/tags/att_a_rel.asp